src/Components/UpdateVotes.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { patchReviewVotes } from "../Utility/api";
+
+const UpdateVotes = ({ votes }) => {
+  const { review_id } = useParams();
+  const [currentVotes, setCurrentVotes] = useState(votes);
+  const [err, setErr] = useState(null);
+  const [disableUpVote, setUpVoteDisable] = useState(false);
+  const [disableDownVote, setDownVoteDisable] = useState(false);
+
+  const handleUpVote = () => {
+    setCurrentVotes((currVotes) => currVotes + 1);
+    setErr(null);
+    handleUpVoteDisable();
+    patchReviewVotes(review_id, 1).catch((err) => {
+      setCurrentVotes((currVotes) => currVotes - 1);
+      setErr("Something went wrong, please try again.");
+    });
+  };
+
+  if (err) return <p>{err}</p>;
+
+  const handleDownVote = () => {
+    setCurrentVotes((currVotes) => currVotes - 1);
+    setErr(null);
+    handleDownVoteDisable();
+    patchReviewVotes(review_id, -1).catch((err) => {
+      setCurrentVotes((currVotes) => currVotes + 1);
+      setErr("Something went wrong, please try again.");
+    });
+    if (err) return <p>{err}</p>;
```

Comment:
this return isn't being returned to anything

Your showing-of-the-error is already being handled when you are setting `setErr` just above here

So you don't need this line at all!